includes/Features/Abilities_Explorer/Abilities_Explorer.php

```diff
@@ -0,0 +1,284 @@
+	 *
+	 * @return bool True if API is available, false otherwise.
+	 */
+	private function check_abilities_api(): bool {
+		// Allow bypassing the check for development/testing
+		if ( defined( 'AI_ABILITIES_EXPLORER_SKIP_API_CHECK' ) && AI_ABILITIES_EXPLORER_SKIP_API_CHECK ) {
+			return true;
+		}
+
+		// Check for WP_Ability class (the official way)
+		if ( class_exists( 'WP_Ability' ) ) {
+			return true;
+		}
+
+		// Check for the main API functions
+		if ( function_exists( 'wp_get_abilities' ) && function_exists( 'wp_register_ability' ) ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	/**
+	 * Display WordPress version notice.
+	 *
+	 * @since 0.1.0
+	 */
+	public function version_notice(): void {
+		$message = sprintf(
+			/* translators: 1: Required WordPress version, 2: Current WordPress version */
+			__( 'Abilities Explorer requires WordPress %1$s or higher. You are running version %2$s.', 'ai' ),
+			self::MIN_WP_VERSION,
+			$GLOBALS['wp_version']
+		);
+
+		printf(
+			'<div class="notice notice-error"><p>%s</p></div>',
+			esc_html( $message )
+		);
+	}
+
+	/**
+	 * Display Abilities API notice.
+	 *
+	 * @since 0.1.0
+	 */
+	public function abilities_api_notice(): void {
+		global $wp_version;
+
+		$message = __(
+			'Abilities Explorer requires the Abilities API which is not available.',
+			'ai'
+		);
+
+		// Build debug information
+		$debug_info   = array();
+		$debug_info[] = sprintf( 'WordPress Version: %s', $wp_version );
+		$debug_info[] = '';
+		$debug_info[] = '=== Required Components ===';
+		$debug_info[] = sprintf( 'WP_Ability class: %s', class_exists( 'WP_Ability' ) ? 'EXISTS ✓' : 'NOT FOUND ✗' );
+		$debug_info[] = sprintf( 'WP_Ability_Category class: %s', class_exists( 'WP_Ability_Category' ) ? 'EXISTS ✓' : 'NOT FOUND ✗' );
+		$debug_info[] = sprintf( 'WP_Abilities_Registry class: %s', class_exists( 'WP_Abilities_Registry' ) ? 'EXISTS ✓' : 'NOT FOUND ✗' );
+		$debug_info[] = '';
+		$debug_info[] = '=== Required Functions ===';
+		$debug_info[] = sprintf( 'wp_register_ability(): %s', function_exists( 'wp_register_ability' ) ? 'EXISTS ✓' : 'NOT FOUND ✗' );
+		$debug_info[] = sprintf( 'wp_get_abilities(): %s', function_exists( 'wp_get_abilities' ) ? 'EXISTS ✓' : 'NOT FOUND ✗' );
+		$debug_info[] = sprintf( 'wp_get_ability(): %s', function_exists( 'wp_get_ability' ) ? 'EXISTS ✓' : 'NOT FOUND ✗' );
+		$debug_info[] = sprintf( 'wp_has_ability(): %s', function_exists( 'wp_has_ability' ) ? 'EXISTS ✓' : 'NOT FOUND ✗' );
+
+		// Check for API version
+		if ( defined( 'WP_ABILITIES_API_VERSION' ) ) {
+			$debug_info[] = '';
+			$debug_info[] = sprintf( 'API Version: %s', WP_ABILITIES_API_VERSION );
+		}
+
+		// Check if the plugin is installed but not activated
+		$debug_info[] = '';
+		$debug_info[] = '=== Installation Status ===';
+		$plugins      = get_plugins();
+		$found_plugin = false;
+		foreach ( $plugins as $plugin_file => $plugin_data ) {
+			if ( stripos( $plugin_data['Name'], 'abilit' ) !== false ) {
```

Comment:
If this is a check if the https://github.com/WordPress/abilities-api plugin is installed, can this not just check for  abilities-api/abilities-api.php array key?

```
		$plugin_file = 'abilities-api/abilities-api.php';
		if ( array_key_exists( $plugin_file, $plugins ) ) {
			$plugin_data  = $plugins[ $plugin_file ];
			$is_active    = is_plugin_active( $plugin_file );
			$debug_info[] = sprintf(
				'Plugin: %s (%s)',
				$plugin_data['Name'],
				$is_active ? 'ACTIVE ✓' : 'INACTIVE ✗'
			);
			$found_plugin = true;
		}
```